packages/ui/src/mocks/helpers/asChainData.ts

```diff
@@ -20,4 +23,9 @@ export const asChainData = (data: any): any => {
   }
 }
 
-const withUnwrap = (data: Record<any, any>) => Object.defineProperty(data, 'unwrap', { value: () => data })
+const withUnwrap = (data: Record<any, any>) =>
+  Object.defineProperties(data, {
+    unwrap: { value: () => data },
+    isSome: { value: Object.keys(data).length > 0 },
+    get: { value: (key: AccountId32) => data[encodeAddress(key.toString())] },
```

Comment:
Good idea but just to make this a bit more general:
```suggestion
    get: {
      value: (key: unknow) => {
        switch (key) {
          case 'object':
            if (key.toRawType?.() === 'AccountId') {
              return data[encodeAddress(key.toString())]
            }
          default:
            return data[key.toString()]
        }
      }
    }
```